add fluentbit mountPath configuration doc (#106)
banzaicloud/logging-operator-docs: docs/configuration/fluentbit.md

  controlNamespace: logging
```

## Log Mount Path

> If the container logs of your k8s cloud provider or deployed k8s environment are not saved in the default directory, you can use the mountPath field to modify. For example:

Reviewer: ```suggestion
If the container logs of your k8s cloud provider or deployed k8s environment are not saved in the default directory, set the `mountPath` field to the path where they are saved. For example:
```
